doc/index.rst

```diff
@@ -1,12 +1,136 @@
+
+Overview
+========
+
+ts_atwhitelight controls the Vera C. Rubin Observatory Auxiliary Telescope white light calibration source.
+
+This light source feeds a monochromator (ATMonochromator), which in turns illuminates the flat field screen.
+
+The white light system is comprised of the following components:
+
+* A KiloArc lamp controller that controls a mercury-vapor lamp with a range 800-1200W. We have two manuals for this: a :download:`user guide <KiloArc Lamp with optional computer control manual rev N.pdf>` and a supplement describing the :download:`remote control option <Remote control option for the KiloArc.pdf>`.
+
+* A ThermoTek T257P chiller. We have two manuals for this: A :download:`user guide <T257P User Manual_Rev X1_02.pdf>` and a detailed description of the :download:`serial communication protocol <TTK Serial Communication Protocol.pdf>`.
+
+* A shutter controlled by a linear actuator.
+
+* A `LabJack T4 <https://labjack.com/products/t4>`_ I/O module to control the lamp controller and linear actuator.
+
+.. _lsst.ts.atwhitelight.preserving_bulb_life:
+
+Preserving Bulb Life
+--------------------
+
+The CSC enforces the following rules, in order to extend the life of these expensive bulbs:
+
+* You may not turn on the lamp unless the chiller is running.
+
+* Warmup phase: the lamp must remain on for at least config.lamp.warmup_duration=900 seconds before you can turn it off without forcing it.
+  This gives the mercury time to fully evaporate.
+  If absolutely necessary you can force the lamp off sooner by issuing the turnLampOff command with force=True.
+  The CSC will force the lamp off and go to fault state if it detects a problem with the chiller or lamp controller.
+
+* Cooldown phase: the lamp must remain off for least config.lamp.cooldown_duration=900 seconds before you can turn it on again.
+  This gives the mercury time to fully recondense.
+
+* You may not turn off the chiller while the lamp is on or cooling down.
+
+In addition, the Operating Parameters section of the manual says the following for the He-Xe bulbs:
+
+* Average bulb lifetime is 1500 hours when operated at 1000 W.
+
+* Operating the lamp at 800 W can increase the bulb lifetime by approximately 5%.
+
+* Operating the lamp at 1200 W can decrease the bulb lifetime by approximately 10%.
+
+* Bulb lifetime will decrease 10 to 20 minutes for each ignition.
+
+.. _lsst.ts.atwhitelight.user_guide:
+
+User Guide
+==========
+
+Start the ATWhiteLight CSC as follows:
+
+.. prompt:: bash
+
+    run_atwhitelight.py
+
+Stop the CSC by sending it to the OFFLINE state.
+
+Turn the lamp on as follows (after you enable the CSC):
+
+* Set a chiller temperature with ``setChillerTemperature``.
+  You may omit this step if the configured ``chiller.initial_temperature`` is appropriate.
+
+* Turn the chiller on with ``startChiller``.
+
+* Wait for the temperature to roughly stabilize, as reported by the ``chillerTemperatures`` telemetry topic.
```

Comment:
Better use disabled/standby/enabled states? So start command to start chiller, enable to turn on the lamp? Or is there any other use for start/enable commands?

Reply:
The basic rule for CSCs that control subsystems is:
- standby: no connection to the subsystems
- disabled: connect to the subsystems but do not control them (except asking for status, if it is not provided automaticallyl)
- enabled: one can now command the subsystem.

We do not want this system to automatically turn on anything: not the chiller, not the white light source. The CSC will probably be enabled most of the time with the lamp off. The lamp is very expensive, has a relatively short life, and runs at 800-1200W. So we only turn it on when absolutely necessary. But it also has to be allowed to cool for a long time before we turn off the chiller. I did lobby for turning the chiller on and off automatically, but was told no: it takes too long for the chiller temperature to stabilize so we can't just turn it on as part of turning the lamp on.